oak-segment-azure/src/main/java/org/apache/jackrabbit/oak/segment/azure/AzureUtilities.java

```diff
@@ -116,6 +116,15 @@ public static CloudBlobDirectory cloudBlobDirectoryFrom(String connection, Strin
 
         return container.getDirectoryReference(dir);
     }
+    public static CloudBlobDirectory cloudBlobDirectoryFrom(String connection, String containerName,
+                                                            String dir,boolean skipCreateContainer) throws InvalidKeyException, URISyntaxException, StorageException {
```

Comment:
Code formatting: for consistency with the rest of the code, please 
* add spaces after commas and  before `{` (in the `if` below) 
* don't align parameter lists with the opening method `(`, a simple indent is sufficient:
```java
public static CloudBlobDirectory cloudBlobDirectoryFrom(String connection, String containerName,
        String dir, boolean skipCreateContainer) throws InvalidKeyException, URISyntaxException, StorageException {
...
}
```